.github/workflows/build-bot-container.yml

```diff
@@ -10,22 +10,25 @@ jobs:
   build-and-publish:
     name: Publish container image
     runs-on: 'ubuntu-latest'
+    permissions:
+      contents: read
+      packages: write
     steps:
```

Comment:
FYI, this is quite important. You are using third-party actions (docker/*) which in principle have a lot of power over your repo (push branches, change issues/PRs, etc.). You hopefully have branch protection rules for `main`, but in general it's a good idea to minimize what workflows can do.

I'd actually recommend to drop the external actions and just run `podman login`, `podman build`, and `podman push`. This isn't really hard, to the contrary: You see what's going on, and these commands can be copy&pasted and ran locally. If you want to, I'm happy to send a PR which does that.

Reply:
Ah right. Hadn't thought about it that way to be honest. Let's drop the external action then.

Reply:
Done in PR #29